cli/src/cli.rs

```diff
@@ -1549,6 +1570,22 @@ fn do_process_ether_deploy(
         config.commitment,
     )?;
 
+    {  // Create code account
+        trace!("Create code account");
+        let make_code_account_instruction = system_instruction::create_account_with_seed(&creator.pubkey(), &program_code, &creator.pubkey(), &program_seed, 10000000, 1024*1024, loader_id);
```

Comment:
Hom many lamports are needed to new account?
Haw many bytes of data to allocate?

Reply:
It depends on the loaded contract. We can get this size from the emulator. But in any case, this changes out of scope current issue.